target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
This is what I have in 0xb00c0000: 

...
RS353aw Bootmonitor V.1.0 from 2013/10/31 00:00:00
Copyright (P) 1996-2013 by Bintec Elmeg GmbH

(1) Boot System
(2) Software Update via TFTP
(3) Software Update via XMODEM
(4) Delete Configuration
(5) Default Bootmonitor Parameters
(6) Show System Information

Your Choice> $

Entering diagnostics mode (Enter e to exit, ? for help)
$ 


(d) Dump Memory (long access)
(b) Dump Memory (byte access)
(t) Set Trace Level
(s) Set System Parameters
(a) Configure Cache
(p) Configure NPC
(c) Show System Configuration
(f) Toggle Fastboot Option
(?) Show Diagnostics Menu
(e) Exit
$ b 


Enter memory address: 0xb00c0000

b00c0000  54 45 4c 44  41 54 20 43  6c 6f 73 65  64 45 79 65  TELDAT ClosedEye
b00c0010  56 69 73 75  61 6c 00 00  2a 02 00 00  00 00 00 00  Visual..*.......
b00c0020  00 03 ff cc  00 fa 05 24  00 00 00 00  31 0d 1d 34  ...L.z.$....1..4
b00c0030  00 00 00 00  1f 8b 08 08  a0 96 52 58  00 03 62 6f  ........ .RX..bo
b00c0040  73 73 2e 62  69 6e 00 ec  fd 0d 78 5c  e5 79 27 0e  ss.bin.l}.x\ey'.
b00c0050  3f 73 66 f4  69 01 c7 f2  58 0c 60 60  46 3a 1a 0d  ?sfti.GrX.``F:..
b00c0060  b6 42 07 a3  24 82 28 61  22 c9 a0 24  4e 33 18 e1  6B.#$.(a"I $N3.a
b00c0070  38 ad 37 55  65 41 dc ac  bb eb 80 bb  71 b7 6e 3a  8-7UeA\,;k.;q7n:
b00c0080  1e c9 44 ed  4e 34 42 38  c8 0e bc d9  a9 fc 81 9b  .IDmN4B8H.<Y)|..
b00c0090  2a 92 01 93  98 5d 17 54  63 88 db 7a  b7 6e 4a 13  *....].Tc.[z7nJ.
b00c00a0  ba 4b 13 95  38 c1 49 d9  d6 69 49 30  c1 70 de df  :K..8AIYViI0Ap^_
b00c00b0  ef 7e 9e 33  73 66 24 13  d2 dd f7 fa  ef 75 bd 7f  o~.3sf$.R]wzou=.
b00c00c0  5f 97 3c 33  e7 3c df cf  fd dc 5f cf  fd 61 2b 65  _.<3g<_O}\_O}a+e
b00c00d0  29 fc eb 59  b3 9e 1f ca  56 81 16 e5  fb b7 39 b7  )|kY3..JV..e{797
b00c00e0  39 a7 fe df  7f ff 7f f3  cf 56 ae fb  ff f6 ff ff  9'~_...sOV.{.v..
b00c00f0  f6 ff 7f 4b  ff a9 cc 38  fe f6 e0 6f  b3 4a d5 0e  v..K.)L8~v`o3JU.
b00c0100  a8 ae 80 ad  3a 62 ea da  40 20 d5 7a  53 ad ba 36  (..-:bjZ@ UzS-:6
b00c0110  35 3a c0 62  73 a9 4c 1f  ca 6c 50 a1  a5 1a 9f a9  5:@bs)L.JlP!%..)
b00c0120  ba fa f9 fc  05 a5 f2 7b  33 91 60 f4  7c 1d 1f 75  :zy|.%r{3.`t|..u
b00c0130  5d 9d 51 ef  fd 93 0c 9f  ad 08 46 55  ad 94 cb 9c  ].Qo}...-.FU-.K.
b00c0140  8d a9 42 26  a6 e6 c6 63  5d b5 ca 7a  77 ed b7 0e  .)B&&fFc]5Jzwm7.
b00c0150  5b 7d a3 cd  f9 ff 94 89  b6 fe db 17  c3 5f f8 b4  [}#My...6~[.C_x4
...

The Bootmonitor expects to find the magic "TELDAT ClosedEyeVisual" followed by length and CRC32, then the gzip-compressed Boss image.